stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/SimpleListAdapter.kt

```diff
@@ -36,5 +36,7 @@ internal abstract class SimpleListAdapter<T : Any, VH : SimpleListAdapter.ViewHo
         val context: Context = itemView.context
 
         abstract fun bind(item: T)
+
+        open fun unbind() {}
```

Comment:
If this is defined here in the base class, the base class should also do the invocation (currently, the inheriting adapters have to do it for themselves)